### PR TITLE
Make database connection persistent in API

### PR DESCRIPTION
**Pull request by Kpaubert**
Make API only have one connection instead of creating one and deleting it on every function call.

### CODE REVIEW

File: backend/api.py [hunk 1]
```diff
@@ -9,6 +9,10 @@
 api = Flask(__name__)
 api.debug = False
 
+database = Database()
```
Comment (AndreasWintherMoen) [inline backend/api.py:12]: Database's constructor does logging, so this must happen after logging is configured.

**Review by AndreasWintherMoen — CHANGES_REQUESTED** [review]
This one seems alright, barring the order of execution. See comment.

**Comment by FredrikAugust** [comment]
Great change! Was going to suggest it just now!

File: backend/api.py (hunk 1 above)
Comment (Kpaubert) [reply backend/api.py]: @AndreasWintherMoen Changed the order of execution so the DB is not instantiated before the logger is configured.

**Review by AndreasWintherMoen — APPROVED** [review]
bitmoji

**Pull request merged**